halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/OrcaService.java

```diff
@@ -77,11 +77,9 @@ protected void appendReadonlyClouddriver(
     profiles.add(profile);
 
     // Plugins
-    String pluginFilename = "plugins.yml";
-    String pluginPath = Paths.get(getConfigOutputPath(), pluginFilename).toString();
     Profile pluginProfile =
-        pluginProfileFactory.getProfile(
-            pluginFilename, pluginPath, deploymentConfiguration, endpoints);
+        pluginProfileFactory.getProfileByName(
```

Comment:
modified to support the new `getProfileByName` method, this should not need updating as we're going with the getProfile method